changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v0.4.1 | 2020-03-23
+* Drop matplotlib < 3 requirement.
+* Add functionality which redirects users to API when accessing particular fields.
+* Add API for markers.
```

Comment:
Definitely needs a very short example.

Reply:
Expanded changelog line and added short tutorial in the docs.
* Add API for markers, i.e. `file.markers` returns a dictionary of markers (see docs tutorials section: Files and Channels).